cubeviz/controls/slice.py

```diff
@@ -122,7 +124,7 @@ def change_slider_value(self, amount):
         new_index = self._slice_slider.value() + amount
         self._slice_slider.setValue(new_index)
 
-        specviz_dispatch.changed_dispersion_position.emit(pos=new_index)
+        # specviz_dispatch.changed_dispersion_position.emit(pos=new_index)
```

Comment:
Is this going to be handled differently now?  If so, can probably delete the commented line.

Reply:
If these commented lines aren't need, maybe best to delete them.